csrankings-g.csv

```diff
@@ -492,6 +492,7 @@ Giorgos Christodoulou 0001,University of Liverpool,https://cgi.csc.liv.ac.uk/~gc
 Giorgos Stamou,NTUA,http://www.image.ntua.gr/~gstam,R3y5dxMAAAAJ
 Giorgos Tolias,Czech Technical University,http://cmp.felk.cvut.cz/~toliageo,e765N80AAAAJ
 Giovanna Rosone,University of Pisa,http://pages.di.unipi.it/rosone,gx_HzA0AAAAJ
+Giovane CM Moura,TU Delft,http://giovane-moura.nl/,DuJtdjUAAAAJ 

```

Comment:
This name does not correspond to either of the DBLP aliases, which require separate entries (https://dblp.org/pid/76/5048.html).

Reply:
You still need to add another entry (Giovane Cesar Moreira Moura ).